### PR TITLE
Added stubs for Simple Linked List exercise

### PR DESCRIPTION
**Pull request by MaruthiMohanReddyPutha**
- Added the stubs for two exercises:

1. Simple Linked List
2. Word Search

### CODE REVIEW

**Comment by MaruthiMohanReddyPutha** [comment]
#705

**Comment by MaruthiMohanReddyPutha** [comment]
Updated the Pull Request to remove the duplicate work.

File: exercises/simple-linked-list/simple-linked-list.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  }
+
+  toArray() {
+    throw new Error("Remove this statement and implement this function");
+  }
+
+  reverse() {
+    throw new Error("Remove this statement and implement this function");
+  }
+
```
Comment (tejasbubane) [inline exercises/simple-linked-list/simple-linked-list.js]: we also need a `head` function which will be a getter. Also `fromArray` & `toArray` are not required. Please refer [spec file](https://github.com/exercism/javascript/blob/master/exercises/simple-linked-list/simple-linked-list.spec.js) for required functions.

File: exercises/simple-linked-list/simple-linked-list.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  add(nextValue) {
+    throw new Error("Remove this statement and implement this function");
+  }
+
+  remove(removeValue) {
+    throw new Error("Remove this statement and implement this function");
+  }
+
+  length() {
```
[the remark this answers is not in the document]
Comment (MaruthiMohanReddyPutha) [reply exercises/simple-linked-list/simple-linked-list.js]: Modified the length() method to a getter.

File: exercises/simple-linked-list/simple-linked-list.js (hunk 1 above)
Comment (MaruthiMohanReddyPutha) [reply exercises/simple-linked-list/simple-linked-list.js]: - Added a new getter property called `head`.
- Removed the `fromArray` method and kept the `toArray` method as it is, as it is required as per the [spec file](https://github.com/exercism/javascript/blob/master/exercises/simple-linked-list/simple-linked-list.spec.js).

**Pull request merged**